force_ext:
- 0.03046106000000215
- 0.028829269999997464
- -5.77428546
pos_fext:
- 0.002584286853043872
- 0.01000603285033993
- 0.07683806366472064
wrench_bias:
- -0.9613222399999998
- 0.79415275
- -0.10982919000000008
- -10.775789670000002
- -6.80065451
- 3.2042039599999987
